spec/models/device_spec.rb

```diff
@@ -2,46 +2,47 @@
 
 RSpec.describe Device, type: :model do
 
-  before do
-    @developer = FactoryGirl::create :developer
-    @device = Device.create(uuid: 1234)
+  let(:developer) { FactoryGirl::create :developer }
+  let(:device) do 
+    dev = FactoryGirl::create(:device)
+    dev.developers << developer
+    dev
+  end
+  let(:checkins) do
```

Comment:
Might be worth keeping this inside the `it "should have some checkins"` block, otherwise these checkins are created for every test which isn't needed